spec/integration/data_stream/manifest.spec.yml

```diff
@@ -208,6 +208,7 @@ spec:
       - logs
       - synthetics
       - traces
+      - profiling
```

Comment:
Btw, I think we should ensure that packages using this type use the latest version of the spec, so we should include this only for packages with format spec >= 2.7.0, for this include something like this in the `versions` section below:
```
  - before: 2.6.0
    patch:
      - op: remove
        path: /properties/type/enum/4
```

Reply:
added in 37f1f69